src/tr2/game/savegame/savegame_legacy.c

```diff
@@ -849,8 +849,9 @@ static bool M_LoadFromFile(MYFILE *const fp)
         Room_SetFlipSlotFlags(i, M_ReadS8() << 8);
     }
 
-    for (int32_t i = 0; i < MAX_MUSIC_TRACKS; i++) {
-        Music_SetTrackFlags(i, M_ReadU16());
+    for (int32_t i = 0; i < LEGACY_MAX_MUSIC_TRACKS; i++) {
```

Comment:
After we remove the dependency on `LEGACY_MAX_MUSIC_TRACKS` in the CDAudio backend, we can make this a module local.